<launch>

<rosparam file="$(find snoopy_setup)/all_parameters.yaml" command="load"/>

<node pkg="snoopy_map" type="ogm_server" name="ogm_server" output="screen"/>

<node pkg="snoopy_pathfind" type="astar" name="astar" output="screen">
	<param name="astar_service" value="astar_srv" />	
</node>

<!-- ASTAR CLIENT PARAMETERS ARE NOT IN all_parameters.yaml THEY ARE SEPARETED -->
<node pkg="snoopy_pathfind" type="astar_client" name="astar_client" output="screen" >
	<param name="pose_x" value="0.2" />
	<param name="pose_y" value="0.25" />
	<param name="cell_occ_value" value="100" />
	<param name="astar_service" value="astar_srv" />
	<param name="grid_map_topic" value="grid_map"/>	
</node>

<node pkg="rviz" name="rviz" type="rviz" args="-d $(find snoopy_pathfind)/path.rviz" />

</launch>
